dmscripts/helpers/brief_data_helpers.py

```diff
@@ -0,0 +1,9 @@
+from datetime import datetime
+from dmutils.formats import DATETIME_FORMAT
+
+
+def get_closed_briefs(data_api_client, date_closed):
```

Comment:
Nice on moving this into a helper

Reply:
Could probably do with a more descriptive name
`data_api_client.find_briefs_iter(status='closed', with_users=True)`
gets closed briefs. This method gets briefs closed on a specific day